gnat2goto/driver/tree_walk.adb

```diff
@@ -792,23 +805,44 @@ package body Tree_Walk is
 
       procedure Handle_Parameter (Formal : Entity_Id; Actual : Node_Id);
 
+      function Handle_Enum_Symbol_Members (Mem : Irep) return Irep;
+      function Handle_Enum_Symbol_Members (Mem : Irep) return Irep is
+         Followed_Type_Symbol : constant Irep :=
+            Follow_Symbol_Type (Get_Type (Mem), Global_Symbol_Table);
+      begin
+         if Kind (Followed_Type_Symbol) = I_C_Enum_Type then
+            declare
+               Val : constant Irep := Global_Symbol_Table
+                 (Intern
+                    (Get_Identifier
+                     (Mem)))
+                 .Value;
+            begin
+               return Val;
+            end;
+         else
+            return Mem;
```

Comment:
So the function is identity for non-`C_Enum_Type`?

Reply:
Yes. Do you think it's better to be documented? I thought it was obvious.

Reply:
No. I was just trying to find out if it was intentional.